src/components/SearchBar.js

```diff
@@ -367,56 +429,45 @@ class SearchBar extends React.Component<SearchBarDefaultProps, SearchBarProps, S
           {advancedMenuItem}
         </Dropdown.Menu>
       </Dropdown>
-    ) : '';
-
-    let placeholder = this.props.placeholder;
-    if (this.props.allowLanguageSelect && language === 'advanced') {
-      placeholder = this.props.placeholderAdvanced;
-    }
+    ) : (
+      ''
+    );
 
-    const suggestionList = this.getSuggestionList();
-    const inputComponent = this.props.autoCompleteUri ?
-      (
-        <AutoCompleteInput
-          uri={`${this.props.baseUri}${this.props.autoCompleteUri}`}
-          updateValue={this.updateQuery}
-          placeholder={placeholder || ''}
-          value={query}
-          className={inputClass}
-        />
-      ) : (
-        <input
-          type="search"
-          className={inputClass}
-          placeholder={placeholder}
-          onChange={this.queryChanged}
-          onKeyDown={this.doKeyPress}
-          value={query}
-        />
-      );
+    const subject = this.props.subject || '';
+    const emailAddress = this.props.email || '';
+    const shareSearch = this.props.allowShareSearch ? (
+      <ShareSearch shareMessage={this.props.shareMessage} subject={subject} email={emailAddress} />
+    ) : (
+      ''
+    );
 
     return (
       <div className={containerClass}>
-        <div className="attivio-globalmast-search" role="search">
+        <div className={subContainerClass} role="search">
           <div className="form-group">
-            {inputComponent}
+            {this.renderInputComponent(query, language)}
             {showMicrophone ? (
               <a onClick={this.startSpeechRecognition} role="button" tabIndex={0}>
                 <span className="attivio-globalmast-search-mic-icon attivio-icon-microphone" style={micStyle} />
               </a>
-            ) : ''}
+            ) : (
+              ''
+            )}
             <button
               type="submit"
               className="btn attivio-globalmast-search-submit"
               onClick={this.doSearch}
-              ref={(c) => { this.submitButton = c; }}
+              ref={(c) => {
+                this.submitButton = c;
+              }}
             >
               {this.props.buttonLabel}
             </button>
           </div>
           {suggestionList}
         </div>
         {languageControl}
+        {shareSearch}
```

Comment:
This can just be:
`{this.props.allowShareSearch && <ShareSearch shareMessage={this.props.shareMessage} subject={subject} email={emailAddress} />}`